nodejs/webapi/server.js

```diff
@@ -0,0 +1,80 @@
+const express = require('express'); // import express
+bodyParser = require('body-parser'); // import body-parser to parse JSON content
+swaggerJSDoc = require('swagger-jsdoc'); // import swagger-jsdoc to generate swagger doc
+swaggerUi = require('swagger-ui-express'); // import swagger-ui-express to serve swagger docs
+const openapi = require('@wesleytodd/openapi'); // import openapi to create openapi spec from json
+const fs = require('fs'); // import fs to read file    
+const path = require('path') // import path to resolve path
+
+// TODO
+const swaggerSpec = require('./swagger/weatherforecast.json')
+// const swaggerSpec = require('./swagger/weatherforecast-reviewed.json')
```

Comment:
Instead of manually reading this, would you be able to use an autogen package like https://www.npmjs.com/package/swagger-autogen ? By doing so, it automatically generates the OpenAPI doc on-the-fly.

Reply:
Thanks Justin. I've updated to use swagger-jsdoc - https://www.npmjs.com/package/swagger-jsdoc/v/4.3.0